src/js/node/_http_agent.ts

```
@@ -1,153 +1,482 @@
-  prototype: Agent;
+function freeSocketErrorListener(err) {
+  const socket = this;
+  $debug("SOCKET ERROR on FREE socket:", err.message, err.stack);
+  socket.destroy();
+  socket.emit("agentRemove");
 }
 
-function Agent(options = kEmptyObject) {
+type Agent = import("node:http").Agent;
+function Agent(options): void {
   if (!(this instanceof Agent)) return new Agent(options);
```

Biome (2.1.2)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)